cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -247,7 +247,11 @@ func (d *driver) nodePrepareResource(ctx context.Context, claim *resourceapi.Res
 		res := kubeletplugin.PrepareResult{
 			Err: fmt.Errorf("error preparing devices for claim %v: %w", claim.UID, err),
 		}
-		return isPermanentError(err), res
+		if isPermanentError(err) {
+			klog.V(6).Infof("Permanent error preparing devices for claim %v: %v", claim.UID, err)
```

Comment:
As I am working on this test, I realized that we didn't emit a log message for the case of emitting a permanent error to the kubelet (we did that indirectly, by having a response-acking log line in the grpc server).

I consider this case happy path from the system's point of view, hence info-level and v6. Example:

>    [pod/nvidia-dra-driver-gpu-kubelet-plugin-5tkrg/compute-domains] I0922 16:17:40.506406       1 driver.go:251] Permanent error preparing devices for claim 97d054d3-5f8b-4826-9a2a-742ef93bc4e6: prepare devices failed: error generating configResultsMap: error getting opaque device configs: error decoding config parameters: strict decoding error: unknown field "unexpectedField"